First draft of JOSS paper (#390)
StingraySoftware/stingray: joss/paper.md

	index: 3
-	name: Department of Astronomy, University of Michigan, 1085 South University Avenue, Ann Arbor, MI 48109, USA
	index: 4
-	name: ESAC/ESA, XMM-Newton Science Operations Centre, Camino Bajo del Castillo s/n, Urb. Villafranca del Castillo, 28692, Villanueva de la Caada, Madrid, Spain
	index: 5
- 	name: Timelab Technologies Ltd., 20-22 Wenlock Road, London N1 7GU, United Kingdom
	index: 6 
- 	name: AinShams University, Egypt
	index: 7
- 	name: Department of Computer Science, North Carolina State University, Raleigh, USA
	index: 8
- 	name: Indian Institute of Technology, Kharagpur West Bengal, India 721302
	index: 9
- 	name: National University of Sciences and Technology (NUST), Islamabad 44000, Pakistan
	index: 10
- 	name: Indian Institute of Technology Mandi, Mandi, Himachal Pradesh, India
	index: 11
date: 26 March 2019
bibliography: joss.bib
---

# Summary

Many celestial objects vary in brightness on timescales of milliseconds to centuries. These ``light curves''--variations of brightness of an object as a function of time--often encode interesting physical processes that can help us learn about the nature of the celestial bodies that produced them. 

> In stars like our sun, typical time scales tell us about stellar rotation, starspots and internal physics like convection. In remnants of stellar explosions like neutron stars, we can use time series to learn about the densest matter known in the universe. Finally, variations in brightness of radiation emitted by gas falling into a black hole gives important clues to the nature of gravity and allows us to test General Relativity to high precision.

Reviewer: ```suggestion
In stars like our sun, typical time scales tell us about stellar rotation, starspots and internal physics like convection. In remnants of stellar explosions like neutron stars, we can use time series to learn about the densest matter known in the universe. Finally, variations in brightness of radiation emitted by gas falling into a black hole give important clues to the nature of gravity and allow us to test General Relativity to high precision.
```
Changed `gives` to `give` and `allows` to `allow`.

Stingray is a modular, class-based library aiming to allow users to build custom workflows for their particular data set and science problem, and implements common operations such as periodograms with standard normalizations, cross spectra and coherence, auto- and cross-correlations, as well as higher-order Fourier products such as bispectra and spectral-timing methods like covariance spectra. The latter consider both time and wavelength of the arriving radiation simultaneously and allows for more comprehensive studies of the underlying physical system. Stingray also implements submodules that allow efficient parametric modelling of periodograms, simulations of realistic time series, and specialized tools to study pulsars. 

> Stingray was designed with a flexible and extensible API to be end-user friendly, but also lies at the core of two other packages: HENDRICS [@hendrics], which implements end-to-end versions of standard data analysis workflows, and DAVE, a graphial user interface designed to enable high-level exploratory data analysis on astronomical time series. A longer publication on the underlying methodology and implementation can be found in [@2019arXiv190107681H], while the source code itself is available on GitHub [@stingraysoftware] as part of a larger ecosystem including tutorials, as well as the repositories for HENDRICS and DAVE.

Reviewer: ```suggestion
Stingray was designed with a flexible and extensible API to be end-user friendly, but also lies at the core of two other packages: HENDRICS [@hendrics], which implements end-to-end versions of standard data analysis workflows, and DAVE, a graphical user interface designed to enable high-level exploratory data analysis on astronomical time series. A longer publication on the underlying methodology and implementation can be found in [@2019arXiv190107681H], while the source code itself is available on GitHub [@stingraysoftware] as part of a larger ecosystem including tutorials, as well as the repositories for HENDRICS and DAVE.
```

Changed `graphial` to `graphical`

# Acknowledgments

> D.H. acknowledges support from the DIRAC Institute in the Department of Astronomy at the University of Washing- ton. The DIRAC Institute is supported through generous gifts from the Charles and Lisa Simonyi Fund for Arts and Sci- ences, and the Washington Research Foundation. M.B. is supported in part by the Italian Space Agency through agree- ment ASI-INAF n.2017-12-H.0 and ASI-INFN agreement n.2017-13-H.0. A.L.S. is supported by an NSF Astronomy and Astrophysics Posdoctoral Fellowship under award AST- 1801792. S.S. was supported by Google Summer of Code 2018. We thank Astro Hack Week for providing the venue that started this project and the Lorentz Center workshop ‘The X- ray Spectral-Timing Revolution’ (February 2016) that started the collaboration. We thank the Google Summer of Code Program for funding a total 6 students who implemented a large fraction of the various library components over three summers.

Reviewer: ```suggestion
D.H. acknowledges support from the DIRAC Institute in the Department of Astronomy at the University of Washing- ton. The DIRAC Institute is supported through generous gifts from the Charles and Lisa Simonyi Fund for Arts and Sci- ences, and the Washington Research Foundation. M.B. is supported in part by the Italian Space Agency through agree- ment ASI-INAF n.2017-12-H.0 and ASI-INFN agreement n.2017-13-H.0. A.L.S. is supported by an NSF Astronomy and Astrophysics Postdoctoral Fellowship under award AST- 1801792. S.S. was supported by Google Summer of Code 2018. We thank Astro Hack Week for providing the venue that started this project and the Lorentz Center workshop ‘The X- ray Spectral-Timing Revolution’ (February 2016) that started the collaboration. We thank the Google Summer of Code Program for funding a total 6 students who implemented a large fraction of the various library components over three summers.
```
Changed `Posdoctoral` to `Postdoctoral`.
